superchain/superchain.go

```diff
@@ -24,7 +24,7 @@ var ErrEmptyVersion = errors.New("empty version")
 //go:embed configs
 var superchainFS embed.FS
 
-//go:embed extra/addresses extra/bytecodes extra/genesis extra/genesis-system-configs
+//go:embed extra/addresses extra/bytecodes extra/genesis
```

Comment:
Can we also drop `extra/addresses`?

Reply:
https://github.com/ethereum-optimism/superchain-registry/pull/386/commits/2a13885ea768ac6d1c2a8b9ed66afcaed9159fa3